plugins/a11ychecker.md

```diff
@@ -161,91 +295,212 @@ If `a11ychecker_allow_decorative_images` is set to `false`, the Accessibility Ch
 
-#### Example: a11ychecker_level
+#### Example: Using `a11ychecker_level`
 
 ```js
 tinymce.init({
-    selector: 'textarea',
-    plugins: 'a11ychecker',
-    toolbar: 'a11ycheck',
-    a11ychecker_level: 'aaa'
+  selector: 'textarea',
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_level: 'aaa'
 });
 ```
 
 {% include misc/plugin-toolbar-button-id-boilerplate.md %}
 
 {% include misc/plugin-menu-item-id-boilerplate.md %}
 
-## API
+## APIs
 
-Accessibility Checker exposes couple of methods that can be called directly.
+The {{pluginname}} plugin provides the following APIs.
 
 ### `toggleaudit()`
 
-Triggers accessibility dialog with the results of the audit and options to correct the problems, if any.
+Opens the accessibility checker dialog with the results of the audit and options to correct the problems, if any.
+
+#### Example: Using `toggleaudit()`
 
-#### Example: toggleaudit()
 ```js
 editor.plugins.a11ychecker.toggleaudit();
 ```
 
 ### `getReport()`
 
-Conducts accessibility audit and reports about the results without triggering the dialog. The report represents an array of issues, each of which has details about:
+Conducts accessibility audit and reports the results without triggering the dialog. The report produced is an array of objects, where each object represents an issue and contains the following details:
 
-* **severity** - *severity of the issue, might be either - info, warning or error*
-* **description** - *brief description of the issue*
-* **url** - *URL of the details page at W3 dedicated specifically to the given issue*
-* **element** - *DOM element having the issue*
+* **id** - The issue identifier used by {{site.productname}}, such as D1, T4A. For a description and other details about the issue, see [Accessibility Rules](#accessiblityrules).
+* **description** - A description of the issue.
+* **severity** - The severity of the issue, either: `info`, `warning`, or `error`.
+* **element** - The DOM element where the issue was found.
+* **url** - A URL to the W3 WCAG technique that needs to be addressed to clear the issue.
 
-##### Example: getReport()
+#### Example: Using `getReport()`
 
 ```js
 var issues = editor.plugins.a11ychecker.getReport();
-```
\ No newline at end of file
+
+console.log(issues);
+
+// example result
+[
+  {
+    "id": "D2",
+    "severity": "error",
+    "url": "http://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html",
+    "description": "Headings must be applied in sequential order. For example: Heading 1 should be followed by Heading 2, not Heading 3.",
+    "element": {}  // The element value contains the DOM element (such as <h4>).
+  },
+  {
+    "id": "T1",
+    "severity": "error",
+    "url": "http://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html",
+    "description": "Tables must have captions",
+    "element": {}  // The element value contains the DOM element (<table>).
```

Comment:
do we care that we have `such as` on the `element` description above but not here?

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1699/commits/57201f8a1b8a53d3c0861a819b68e45f66e68798